From acb44997acc886bf61b054fe1e18a23a28ea899b Mon Sep 17 00:00:00 2001
From: Balazs Nagy <julsevern@gmail.com>
Date: Tue, 9 Feb 2010 18:22:29 +0100
Subject: [PATCH 2/3] Typo fix in test

If publisher_magazines_path(1) renders /publishers/1/magazines,
/magazines/2/photos should be generated from magazine_photos_path(2)
instead of magazines_photos_path(2)

Signed-off-by: Balazs Nagy <julsevern@gmail.com>
---
 actionpack/test/dispatch/routing_test.rb |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/actionpack/test/dispatch/routing_test.rb b/actionpack/test/dispatch/routing_test.rb
index 6f77281..8c4dd96 100644
--- a/actionpack/test/dispatch/routing_test.rb
+++ b/actionpack/test/dispatch/routing_test.rb
@@ -732,7 +732,7 @@ class TestRoutingMapper < ActionDispatch::IntegrationTest
       assert_equal '/publishers/1', publisher_path(1)
       assert_equal '/publishers/1/magazines', publisher_magazines_path(1)
       assert_equal '/magazines/2', magazine_path(2)
-      assert_equal '/magazines/2/photos', magazines_photos_path(2)
+      assert_equal '/magazines/2/photos', magazine_photos_path(2)
       assert_equal '/photos/3', photo_path(3)
     end
   end
-- 
1.6.6.1

